src/main/resources/db/migration/V1_21__activity.sql

```diff
@@ -0,0 +1 @@
+alter table activity_schedule_mapping add column when_updated timestamp with time zone;
```

Comment:
does this need a default otherwise existing rows can't be migrated?
Does it need to be nullable?
Why are you storing the value with a time zone and then using a `LocalDateTime`?

Reply:
I agree with the time zone but I've just copied the existing conventions in the project.

isn't it nullable now - missing `not null`. I thought it would default to null anyway, didn't realise I have to be explicit.